ExportKindleClippingsToNotion.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Muss das ins Repo? Sollte doch eher in die gitignore oder?

Reply:
Ne Solution kannst du dir wie eine Mappe um mehrere dotnet Projekte vorstellen. Sollte eigentlich immer vorhanden sein, auch wenn du nur ein Projekt hast. Gehört auch ins Repo.

In der Regel ist die .sln Datei, die Datei, die man mit der IDE öffnet.